`initial_language_from_accept_language_header` requires axum

[The official axum workspace starter](https://github.com/leptos-rs/start-axum-workspace) splits the project into three packages: server, frontend, and app. Only the server package directly depends on axum. To my understanding, `leptos-fluent` should be added to `app/Cargo.toml` in the workspace setup, which works fine until we set `initial_language_from_url_path_to_server_function` to true:

```
error[E0433]: failed to resolve: could not find `axum` in the list of imported crates
  --> app/src/lib.rs:66:5
   |
66 | /     leptos_fluent! {
67 | |         children: children(),
68 | |         locales: "locales",
69 | |         default_language: "en",
...  |
78 | |         initial_language_from_url_path_to_server_function: true,
79 | |     }
   | |_____^ could not find `axum` in the list of imported crates
```

because the app package does not directly depend on axum.

I solved this by adding axum as an optional dependency in `app/Cargo.toml` and enabling it when building with the ssr feature, but I'm not sure if this is the proper way (I'm new to this) or if this should be done by the `"leptos-fluent/axum"` in `app/Cargo.toml`. If the former, I think this should be documented at https://mondeja.github.io/leptos-fluent/latest/install.html#ssr

--- a/README.md
+++ b/README.md
@@ -22,14 +22,16 @@ Add the following to your `Cargo.toml` file:
 ```toml
 [dependencies]
 leptos-fluent = "0.2"
+axum = { version = "0.8", optional = true }
 
 [features]
 hydrate = [
-  "leptos-fluent/hydrate"
+  "leptos-fluent/hydrate",
 ]
 ssr = [
   "leptos-fluent/ssr",
-  "leptos-fluent/actix",  # actix and axum are supported
+  "leptos-fluent/axum",  # actix and axum are supported
+  "dep:axum",
 ]
 ```
 
@@ -240,7 +242,6 @@ fn LanguageSelector() -> impl IntoView {
 - **JSON5 languages file**: `json5`
 - **Tracing support**: `tracing`
 - **Debugging**: `debug`
-- **Disable Unicode isolating marks**: `disable-unicode-isolating-marks`
 
 ## Resources
 

--- a/book/src/install.md
+++ b/book/src/install.md
@@ -6,7 +6,7 @@
 
 ## CSR
 
-For client side rendering apps install **leptos-fluent**:
+For client side rendering apps just install **leptos-fluent**:
 
 ```toml
 [dependencies]
@@ -22,14 +22,16 @@ features set.
 ```toml
 [dependencies]
 leptos-fluent = "0.2"
+axum = { version = "0.8", optional = true }
 
 [features]
 hydrate = [
-  "leptos-fluent/hydrate"
+  "leptos-fluent/hydrate",
 ]
 ssr = [
   "leptos-fluent/ssr",
   "leptos-fluent/axum",  # actix and axum are supported
+  "dep:axum",
 ]
 
 # Using cargo-leptos
@@ -40,8 +42,8 @@ watch-additional-files = ["locales"]
 ## Desktop applications
 
 **leptos-fluent** can be installed on non-wasm targets, like desktop
-applications. You need to install `leptos-fluent`, [`fluent-templates`]
-and enable the `system` feature:
+applications. You need to install **leptos-fluent**, and enable the
+`system` feature:
 
 ```toml
 [dependencies]
@@ -61,7 +63,6 @@ leptos-fluent = { version = "0.2", features = ["system"] }
 - **JSON5 languages file**: `json5`
 - **Tracing support**: `tracing`
 - **Debugging**: `debug`
-- **Disable Unicode isolating marks**: `disable-unicode-isolating-marks`
 
 ## Nightly toolchain
 
@@ -121,7 +122,6 @@ leptos-fluent = { version = "0.2", features = ["tracing"] }
 fluent-templates = "0.13"
 ```
 
-[`fluent-templates`]: https://github.com/XAMPPRocky/fluent-templates
 [`cargo leptos`]: https://github.com/leptos-rs/cargo-leptos
 [`tracing`]: https://docs.rs/tracing/latest/tracing
 

--- a/book/src/languages.md
+++ b/book/src/languages.md
@@ -115,17 +115,9 @@ order will used as the initial of the user when no other initialization
 language is discovered.
 
 ```rust
-static_loader! {
-    static TRANSLATIONS = {
-        locales: "./locales",
-        fallback_language: "en",
-    };
-}
-
 #[component]
 pub fn App() -> impl IntoView {
     leptos_fluent! {
-        translations: [TRANSLATIONS],
         languages: "./locales/languages.json5",
         default_language: "en",
     };

--- a/book/src/leptos_fluent.md
+++ b/book/src/leptos_fluent.md
@@ -198,23 +198,13 @@ language when a translation is not found in the current language.
 ### `core_locales`
 
 Common locale resources that are shared across all locales.
-Must be relative to the _Cargo.toml_ file, the same
-used in the [`fluent_templates::static_loader!`] macro:
+Must be relative to the _Cargo.toml_ file:
 
 ```rust
-static_loader! {
-    pub static TRANSLATIONS = {
-        locales: "./locales",
-        core_locales: "./locales/core",
-        fallback_language: "en",
-    };
-}
-
 leptos_fluent! {
     locales: "./locales",
     core_locales: "./locales/core",
     // ^^^^^^^^^^^^^^^^^^^^^^^^^^^
-    translations: [TRANSLATIONS],
 }
 ```
 

--- a/leptos-fluent/README.md
+++ b/leptos-fluent/README.md
@@ -22,14 +22,16 @@ Add the following to your `Cargo.toml` file:
 ```toml
 [dependencies]
 leptos-fluent = "0.2"
+axum = { version = "0.8", optional = true }
 
 [features]
 hydrate = [
-  "leptos-fluent/hydrate"
+  "leptos-fluent/hydrate",
 ]
 ssr = [
   "leptos-fluent/ssr",
-  "leptos-fluent/actix",  # actix and axum are supported
+  "leptos-fluent/axum",  # actix and axum are supported
+  "dep:axum",
 ]
 ```
 
@@ -240,7 +242,6 @@ fn LanguageSelector() -> impl IntoView {
 - **JSON5 languages file**: `json5`
 - **Tracing support**: `tracing`
 - **Debugging**: `debug`
-- **Disable Unicode isolating marks**: `disable-unicode-isolating-marks`
 
 ## Resources
 

--- a/leptos-fluent/src/lib.rs
+++ b/leptos-fluent/src/lib.rs
@@ -21,14 +21,16 @@
 //! ```toml
 //! [dependencies]
 //! leptos-fluent = "0.2"
+//! axum = { version = "0.8", optional = true }
 //!
 //! [features]
 //! hydrate = [
-//!   "leptos-fluent/hydrate"
+//!   "leptos-fluent/hydrate",
 //! ]
 //! ssr = [
 //!   "leptos-fluent/ssr",
-//!   "leptos-fluent/actix",  # actix and axum are supported
+//!   "leptos-fluent/axum",  # actix and axum are supported
+//!   "dep:axum",
 //! ]
 //! ```
 //!
@@ -239,7 +241,6 @@
 //! - **JSON5 languages file**: `json5`
 //! - **Tracing support**: `tracing`
 //! - **Debugging**: `debug`
-//! - **Disable Unicode isolating marks**: `disable-unicode-isolating-marks`
 //!
 //! # Resources
 //!

